src/smc-webapp/app-framework/Store.ts

```diff
@@ -107,7 +111,7 @@ export class Store<State> extends EventEmitter {
     notSetValue?: any
   ): any => {
     if (this.selectors && this.selectors[field] != undefined) {
-      return this.selectors[field].fn();
+      return this.selectors[field].fn(this.getState());
```

Comment:
`reselect` memoizes the return function based on the args so we have give a changing value here. Upside is that if `getState` doesn't change, no recomputes will happen.